File: C_Programs/PrintAvgSalaryOf10UsingForLoop.cpp
//Accept salary of 10 and print average salary using for loop

#include<stdio.h>
#include<conio.h>

int main()
{
	int cnt,salary,total=0;
	float avg;
	for(cnt=1;cnt<=10;cnt++)
	{
		printf("\nEnter the Salary of %d Person : ",cnt);
		scanf("%d",&salary);
		total=total+salary;
	}
	avg=(float)total/10;
	printf("\nAverage Salary : %f",avg);
	return 0;
}

File: Core_Java_Programs/PrintAvgSalaryOf10UsingForLoop.java
//Accept salary of 10 and print average salary using for loop
import java.util.*;

public class PrintAvgSalaryOf10UsingForLoop {
    public static void main(String[] args) {
        int cnt,salary,total=0;
        float avg;
        Scanner sc=new Scanner(System.in);

        for(cnt=1;cnt<=10;cnt++)
        {
            System.out.println("Enter the Salary of "+cnt+" Person : ");
            salary=sc.nextInt();
            total=total+salary;
        }
        avg=(float)total/10.0f;
        System.out.println("Average Salary : "+avg);
    }
}
